### ISSUE TITLE
Feature Request: Enable plugin functionality for other themes

### ISSUE THREAD

**Issue by greatislander** [BCcampus/pressbooks-textbook#158]
### Description 

Currently, users must enable the Open Textbooks theme on their books for much of the functionality of this plugin to become available to them. With a few exceptions noted below, this should not be a requirement. Users may want to take advantage of the features of Textbooks for Pressbooks while using a different book theme.

**Expected behaviour** 

All features of Textbooks for Pressbooks are available regardless of which book theme is selected.

**Actual behaviour** 

Many features are tightly coupled to the Open Textbooks theme.

### Notes

As far as I can tell, changes to the web UI of the book theme are the only components of Textbooks for Pressbooks that would require the use of a custom theme. There may be [other ways we could approach this](https://github.com/pressbooks/snippets/tree/master/webbook-theme-hooks) and I'm happy to help.

### System Information

- Textbooks for Pressbooks: 4.0.3 
- Pressbooks: 5.2.1 
- WordPress: 4.9.5

**Comment by bdolor** [comment]
I agree with this approach @greatislander With the recent changes that you've introduced to PB and the flexibility/stability that comes with that I think the time is right to be looking at breaking the plugin features away from theme dependencies.

**Comment by greatislander** [comment]
Thanks @bdolor, glad to hear we are thinking similarly about this. In terms of support for this change from the Pressbooks end, feel free to tag me in if you need anything.
